spec/controllers/bets_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe BetsController, type: :controller do
+  describe 'GET #index' do
+    subject { get :index }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'GET #show' do
+    let(:user) do
+      User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+    end
+    let(:bet) do
+      Bet.create(bet: 'Bet text',
+                 expires_at: DateTime.tomorrow,
+                 creator_id: user.id)
+    end
+    subject { get :show, params: { id: bet.id } }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'GET #new' do
+    subject { get :new }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'POST #create' do
+    let(:user) do
+      User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+    end
+    let(:valid_params) do
+      { bet: 'Bet text',
+        expires_at: DateTime.tomorrow,
+        creator_id: user.id }
+    end
+    let(:invalid_params) do
+      { bet: 'Invalid Bet',
+        expires_at: DateTime.yesterday,
+        creator_id: user.id }
+    end
+    subject { post :create, params: { bet: bet_params } }
```

Comment:
This is a style comment, but I prefer the subject to be the first thing in the test, so its easy to find exactly what is being tested.

```ruby 
describe 'thing' do
  subject { ... }
end
```